src/game/frontend/menu/Menu.cpp

```diff
@@ -28,6 +30,24 @@ namespace YimMenu
 				LOG(INFO) << coords.x << "x\t" << coords.y << "y\t" << coords.z << "z";
 			}
 
+			if (ImGui::Button("Spawn Ped"))
+			{
+				LOG(INFO) << "Before spawn";
+				FiberPool::Push([] {
+					auto model_hash = "U_F_M_RHDNudeWoman_01"_J; // most models don't work
```

Comment:
Possible because this ped doesn't have any clothes or other stuff on them?

Reply:
Testing in SP, 5 different models spawned with no issues. Even MP models.

Reply:
Post the models that you have issues spawning